rosa_sts_managed_oidc_no_modules/terraform.tfvars

```diff
@@ -0,0 +1,38 @@
+#AWS Info
+aws_region="eu-west-1"
+availability_zones = ["eu-west-1a","eu-west-1b","eu-west-1c"]
+
+# Create a VPC?
+create_vpc=true
+vpc_name="poc-andyr1"
```

Comment:
Same question here, do we want to specify non-user specific vars?

Reply:
Nope, I'll fix this